openpgp.d.ts

```diff
@@ -610,38 +667,14 @@ export function armor(messagetype: enums.armor, body: object, partindex: number,
 /**
  * DeArmor an OpenPGP armored message; verify the checksum and return the encoded bytes
  */
-export function unarmor(text: string): object;
-
-
-/** Class that represents an OpenPGP cleartext signed message.
- */
-export class CleartextMessage {
-  /** Returns ASCII armored text of cleartext signed message
-   */
-  armor(): string;
-
-  /** Returns the key IDs of the keys that signed the cleartext message
-   */
-  getSigningKeyIds(): Array<Keyid>;
-
-  /** Get cleartext
-   */
-  getText(): string;
-
-  /** Sign the cleartext message
-   *
-   *  @param privateKeys private keys with decrypted secret key data for signing
-   */
-  sign(privateKeys: Array<Key>): void;
-
-  /** Verify signatures of cleartext signed message
-   *  @param keys array of keys to verify signatures
-   */
-  verify(keys: Key[], date?: Date, streaming?: boolean): Promise<VerificationResult[]>;
+export function unarmor<T extends MaybeStream<Uint8Array>>(text: string): Promise<{ text: string, data: T, type: enums.armor }>;
```

Comment:
I think this template and `Maybe` are unnecessary:
```suggestion
export function unarmor(text: string): Promise<{ text: string, data: Stream<Uint8Array>, type: enums.armor }>;
```